src/core/agent/services/ipexCommunicationService.ts

```diff
@@ -95,7 +95,8 @@ class IpexCommunicationService extends AgentService {
     await this.admitIpex(
       notifRecord.a.d as string,
       holder.signifyName,
-      exn.exn.i
+      exn.exn.i,
+      exn.exn.e.acdc.s
```

Comment:
What about for chained credentials? I think we need a list of schemas

Reply:
Are we going to take all the schema SAIDs (currently 3) and resolve all of them here?

Reply:
yep